contributors/devel/api-conventions.md

```diff
@@ -306,34 +306,57 @@ response reduces the complexity of these clients.
 ##### Typical status properties
 
 **Conditions** represent the latest available observations of an object's
-current state. Objects may report multiple conditions, and new types of
-conditions may be added in the future. Therefore, conditions are represented
-using a list/slice, where all have similar structure.
+state.  They are an extension mechanism intended to be used when the details of
+an observation are not a priori known or would not apply to all instances of a
```

Comment:
I list "ready" as an example a couple lines later.  Not sufficient?

Reply:
thanks @thockin we were basically writing a operator and struggling to find some guidelines on when to use conditions in it when i stumbled upon this.  The `ready` example below is mainly an example of what should not be a condition since it applies to all instances of the Pod. I am mainly trying to decipher this line `when the details of an observation are not a priori known` for an example of a condition/observation to which that applies.   Thanks for updating the guidelines.